modules/agent-policy/main.tf

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "gcloud" {
+  source = "terraform-google-modules/gcloud/google"
+
+  platform              = "linux"
+  additional_components = ["alpha"]
+
+  create_cmd_entrypoint = abspath("${path.module}/scripts/create-update-script.sh")
+  create_cmd_body       = "${var.project_id} ${jsonencode(var.policy_id)} ${jsonencode(var.description == null ? "" : var.description)} ${base64encode(jsonencode(var.agent_rules))} ${base64encode(jsonencode(var.group_labels == null ? [] : var.group_labels))} ${base64encode(jsonencode(var.os_types))} ${base64encode(jsonencode(var.zones == null ? [] : var.zones))} ${base64encode(jsonencode(var.instances == null ? [] : var.instances))}"
```

Comment:
I remember trying it, and none of the arguments were passed into the bash script, but I think I just figured it out! Because `create_cmd_body` is the arg list that's passed into the bash script, I need to put `\` at the end of each line between `<<EOT and EOT` to pass in all the args in the arg list.